src/gui_common/AddWindowReorderingSupportToSiblings.cs

```diff
@@ -394,6 +399,32 @@ private void OnWindowReorder(CustomDialog window)
         window.SetAsToplevel(isSetAsToplevel);
     }
 
+    private void OnWindowOpen(CustomDialog window)
+    {
+        // Tell the system that there is an opened window and wait in case more windows will be opened at once.
+        if (openedWindows.Count == 0)
+            CallDeferred(nameof(ReorderOpenedWindows));
+
+        openedWindows.Add(window);
+    }
+
+    private void ReorderOpenedWindows()
+    {
+        // Sort the windows to make sure they are updated in the right order.
+        openedWindows.Sort(delegate(CustomDialog first, CustomDialog second)
```

Comment:
This makes a lambda, right? The syntax with the delegate keyword should not be used. Instead the `=>` syntax should be used.